src/Driver/BlueimpUploadDriver.php

```diff
@@ -117,7 +115,6 @@ public function download(Request $request, StorageConfig $config)
 
     /**
      * @param \Illuminate\Http\Request $request
-     * @param \LaraCrafts\ChunkUploader\Identifier\Identifier $identifier
      * @param StorageConfig $config
```

Comment:
Could you write this out fully?

Reply:
I changed this but while thinking about this the following came to my mind:
If the type is `StorageConfig` then the type of the `config` is already defined. In this case, do we need to define the name fully or the type of the variable already contains this information?

Reply:
(In my opinion) we should always write the name out fully, even if it's already imported

Reply:
In that case I misunderstood you.